galley/pkg/kube/source/listener.go

```diff
@@ -152,19 +153,25 @@ func (l *listener) handleEvent(c resource.EventKind, obj interface{}) {
 	if !ok {
 		var tombstone cache.DeletedFinalStateUnknown
 		if tombstone, ok = obj.(cache.DeletedFinalStateUnknown); !ok {
-			scope.Errorf("error decoding object, invalid type: %v", reflect.TypeOf(obj))
+			err := fmt.Sprintf("error decoding object, invalid type: %v", reflect.TypeOf(obj))
```

Comment:
Can you rename `err` to `msg`. `err` conventionally implements the [`error` interface](https://golang.org/pkg/builtin/#error).

Reply:
Done.